app/serializers/basic_endpoint_serializer.rb

```diff
@@ -2,9 +2,12 @@
 # Endpoint serializer returning only basic information.
 #
 class BasicEndpointSerializer < ActiveModel::Serializer
+  include Vphshare::Application.routes.mounted_helpers
+
   attributes :id, :name, :description, :endpoint_type, :url
 
   def url
-    atmosphere.descriptor_api_v1_endpoint_url(object.id)
+    atmosphere.
+      descriptor_api_v1_endpoint_url(object.id)
```

Comment:
Right, I will revert it. It was done during active model serializer 0.8.3 tests.